src/main/resources/templates/profile.html

```
@@ -50,28 +55,67 @@
+    <div class="report-column">
+        <div class="card">
+            <div class="card-header">
+                <span class="card-kicker">Konto uppgifter</span>
+                <label for="enableEdit" class="card-kicker">
+                    <input type="checkbox" id="enableEdit"> Redigera profil
+                </label>
+            </div>
+            <div class="field-grid">
+                <form id="profileForm" th:object="${updateProfile}" th:action="@{/profile/update}" method="post">
+                    <input type="hidden" name="id" th:value="*{id}">
+                    <input th:field="*{fullname}" type="text" disabled></input>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </input> ]

(tag-pair)